fsharp-backend/src/BackendOnlyStdLib/StdLib.fs

```diff
@@ -1,16 +1,16 @@
+/// Functions that can only be run on the backend
+///
+/// Aggregates functions in other modules
 module BackendOnlyStdLib.StdLib
 
-// Functions that can only be run on the backend
-
 // CLEANUP - some of these functions can be run on the client too after the switch to
 // F#, esp LibJwt, LibCrypto and LibX509, plus at least some of LibHttpClient*
 
-open Prelude
-
 module RT = LibExecution.RuntimeTypes
 
 let fn = RT.FQFnName.stdlibFnName
 
+// WHATISTHIS
```

Comment:
In the ocaml version, we renamed functions by copying them. But that led to more code, and it was tough to see if they were identical or not.

Instead, I decided just keep one implementation (named according to its latest name), and a list of renames. The left is the old function, the right is the new function name. See RuntimeTypes.renameFunctions